docs/semgrep-ci/ci-environment-variables.md

```diff
@@ -205,7 +205,9 @@ pipelines:
 
 ### `SEMGREP_REPO_NAME`
 
-Set `SEMGREP_REPO_NAME` to define the repository name used to construct URLs for calls to your source code manager's external API to publish review comments on pull requests or merge requests. To avoid hardcoding this value, check your CI provider's documentation for available environment variables that can automatically detect the correct values for every CI job. 
+Set `SEMGREP_REPO_NAME` to create a repository name when scanning with a CI provider we don't provide support for. For hyperlinks and PR comments to work, this name should be the same as your repository name understood by your CI provider.
+
+To avoid hardcoding this value, check your CI provider's documentation for available environment variables that can automatically detect the correct values for every CI job. 
 
```

Comment:
```suggestion

Semgrep automatically detects `SEMGREP_REPO_NAME` if your [provider is listed in Semgrep Cloud Platform](/deployment/add-semgrep-to-ci/#guided-setup-for-ci-providers-in-scp). In this case, there is no need to set the variable.
```
Hi @emjin! You can choose to add this line per @bkettle's suggestion if you so choose. It's a little wordy, but that being said, sometimes people have a hard time discerning which env vars to set so I tend to err on spelling things out. I'd say 30-40% of the community questions are about CI providers.)